packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -232,6 +232,10 @@ $search-transition-timing: 95ms;
       outline: none;
     }
   }
+
+  .#{$prefix}--masthead__nav-item li {
+    height: 100%;
```

Comment:
Trying to understand why this was added here?

Reply:
If I'm understanding correctly, think if you fix the invalid HTML above you won't need this.